### ISSUE TITLE
Obvious over subtraction of background and strange "fence" features in the vicinity of large galaxies in mosaic3 (90prime) z-band images

### ISSUE THREAD

**Issue by siyue** [moustakas/SGA#59]
Many z-band images of nearby galaxies have some "fence" patterns throughout the galaxy. Below should you one example NGC2742:
<img width="652" alt="截屏2022-08-21 12 02 25" src="https://user-images.githubusercontent.com/8948338/185785872-f70cd7a4-654f-4cb2-8557-bf7283a4632a.png">
Meanwhile, the sky background is over subtracted:
<img width="596" alt="截屏2022-08-21 12 06 29" src="https://user-images.githubusercontent.com/8948338/185786000-0fd0e25d-3f2a-45c3-af37-642930b619b0.png">
The region enclosed by the green curves are obviously (for me) darker than other sky regions! 
I didn't find the same problem in the g-band, r-band, and DECam z-band data. 
This problem may make the photometry based on MzLS z-band data suspicious.
Is the DESI team aware of the problem and are there any solutions?
Otherwise I have to remove all the galaxies observed with  MzLS from my sample. very upset.

**Comment by dstndstn** [comment]
Yes, we're very aware of this issue.  The Mosaic3 camera has pattern noise, and these artifacts arise from the algorithm that tries to remove it.

**Comment by siyue** [comment]
Thanks! just learn more about this infuriating noise! Then I will remove all these galaxies from my sample. If I may make a comment, I would suggest making this potential problem explicit in the SGA's description!

**Comment by siyue** [comment]
YES! They are too green!! This is also how I found the problem!!
